deploy/kube-config/influxdb/grafana-service.yaml

```diff
@@ -2,16 +2,17 @@ apiVersion: v1
 kind: Service
 metadata:
   labels:
-    kubernetes.io/cluster-service: 'true'
```

Comment:
@aegixx Well, I guess that the "best-practice" approach would be to deploy it as a cluster-service. It handles update/restart/shutdown as opposed to a manual deployment. As long as it's clear in the docs (we can work on that part!) I feel like a cluster service is a best setup.

In the old configs, influxdb and grafana are marked as cluster services. Since Chronograf is essentially a Grafana replacement, it should also be marked as one.